CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ endif()
 # - npm/package.json
 # - hermes.podspec
 project(Hermes
-        VERSION 0.5.2
+        VERSION 0.5.3
         LANGUAGES C CXX)
 # Optional suffix like "-rc3"
 set(VERSION_SUFFIX "-rc1")
```

Comment:
Oops, sorry for not mentioning this `"-rc1"` need to be reset to `""` as well, otherwise artifacts would be named with `-rc1` hence the current CircleCI failures. It was set in https://github.com/facebook/hermes/commit/c9d1d8335e8f34242a26646665228d522b7df2ae.

Reply:
Ah my bad, updated 👍